css/components/images.css

```diff
@@ -0,0 +1,13 @@
+.img-responsive {
```

Comment:
This PR is for avatars and badges right? Didn't get to see the images related documentation and demo part yet. So you can check it

Reply:
I added the image css class since I needed to use it for avatar, and will be reusing it for images as well